src/options/options_handler.cpp

```diff
@@ -1368,55 +1370,66 @@ void OptionsHandler::notifySetPrintExprTypes(std::string option) {
 
 
 // main/options_handlers.h
+
+#define PRINT_CONFIG(str,cond) \
```

Comment:
Why is this a macro and not just a function? Why use printf instead of std::cout?

Reply:
Because it's simpler, and in this particular case a function is almost overkill. But I can make a function out of it if you prefer.
Because it was a printf before.

Reply:
Got rid of printf in options_handler.cpp and did some additional cleanup.